packages/base/skill-card.gts

```diff
@@ -1,11 +1,66 @@
+import {
+  CardDef,
+  Component,
+  FieldDef,
+  field,
+  contains,
+  containsMany,
+} from './card-api';
+import BooleanField from './boolean';
+import CodeRefField from './code-ref';
 import MarkdownField from './markdown';
-import { CardDef, Component, field, contains } from './card-api';
+import StringField from './string';
 import RobotIcon from '@cardstack/boxel-icons/robot';
 
+function djb2_xor(str: string) {
+  let len = str.length;
+  let h = 5381;
+
+  for (let i = 0; i < len; i++) {
+    h = (h * 33) ^ str.charCodeAt(i);
+  }
+  return (h >>> 0).toString(16);
+}
+
+function friendlyModuleName(fullModuleUrl: string) {
+  return fullModuleUrl
+    .split('/')
+    .slice(-1)[0]
+    .replace(/\.gts$/, ' ');
```

Comment:
seems a little strange, why are we replacing the extension with a space character. isn't the goal to remove the extension? Also, `.pop()!` might be a little more easy to read than `.slice(-1)[0]`

Reply:
I think the space is a mistake